### PR TITLE
Added tests to verify host volume for minio

### LINKED ISSUE #34: Add tests for host volume feature
Add tests for host volume feature:
Tests:

**Scenario 1**

1. Deploy minio
2. Write some data
3. Destroy the job
4. Deploy new minio with existing volume
5. Verify that data available on new instance

**Scenario 2**

1. Deploy minio
2. Write some data
3. Simulate container crash (docker cli, stop container and remove it)
4. Nomad should automatically create a new instance of container with the same volume
5. Verify that data available on new instance

### PR DESCRIPTION
**Pull request by Neha-Sinha2305**
closes #34

### CODE REVIEW

File: dev/ansible/04_upload_files.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@
   tags: example-upload
 
 - name: Verify that file exist on target `/one/some/prefix/iris-file.csv`
-  shell: mc cat minio-container/one/some/prefix/iris-file.csv
+  shell: mc ls minio-container/one/some/prefix/iris-file.csv
```
Comment (zhenik) [inline dev/ansible/04_upload_files.yml:80]: Are you sure that you can run `ls` on object(`iris-file.csv`), I thought `ls` is only for buckets and directories under bucket? [doc](https://docs.min.io/docs/minio-client-complete-guide.html#ls)

Comment (Neha-Sinha2305) [reply dev/ansible/04_upload_files.yml:80]:
```
ls          list buckets and objects
cat         display object contents
```
I guess object means the same for both **ls** and **cat** commands. I checked the o/p by adding a debug message and it worked, i can add debug messages so as we have clarity based on the o/p displayed.

**Review by zhenik — APPROVED** [review]

File: dev/ansible/04_upload_files.yml [hunk 2]
```diff
@@ -58,19 +56,17 @@
...
 
 - name: Verify that file exist on target `/one/some/prefix/iris-file.csv`
-  shell: mc cat minio-container/one/some/prefix/iris-file.csv
-  failed_when: csv_copy.rc == 1
+  shell: mc ls minio-container/one/some/prefix/iris-file.csv
   tags: example-upload
 
-- name: Upload `json` examples to target `/one/prefix/json/`
+- name: Upload data to target `/one/data/`
   shell: mc cp --recursive ../../example/resources/data/ minio-container/one/data/
```
Comment (pdmthorsrud) [inline dev/ansible/04_upload_files.yml:86]:
```suggestion
  shell: mc cp --recursive ../../example/resources/data/minio-container/one/data/
```

File: dev/ansible/05_test_persistence.yml [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+- name: Fetch bootstrap token nomad
+  shell:
+    cmd: vault kv get -field=secret-id secret/nomad-bootstrap-token
+  when: lookup('env', 'nomad_acl') | bool
+  register: bootstrap_token
+
+- name: Verify if host volume has been added on nomad node(acl enabled)
+  shell:
+    cmd: nomad node status -short -self | grep 'Host Volumes' | tr -d " \t\n\r"  | cut -d "=" -f2
+  register : nomad_node_status
```
Comment (pdmthorsrud) [inline dev/ansible/05_test_persistence.yml]:
```suggestion
  register: nomad_node_status
```

File: dev/ansible/04_upload_files.yml (hunk 2 above)
Comment (zhenik) [reply dev/ansible/04_upload_files.yml:86]: @pdmthorsrud I think it was correct. 
`mc cp --recursive <source> <target>`[doc](https://docs.min.io/docs/minio-client-complete-guide.html#cp)

Comment (pdmthorsrud) [reply dev/ansible/04_upload_files.yml:86]: ... oops. I'm a dumbdumb :D Sorry!

**Review by pdmthorsrud — APPROVED** [review]

**Pull request merged**